mne/surface.py

```diff
@@ -423,11 +439,21 @@ def read_surface(fname, verbose=None):
     tris : int array, shape=(n_faces, 3)
         Triangulation (each line contains indexes for three points which
         together form a face).
-
+    volume_info : dict-like
+        If read_metadata is true, key-value pairs found in the geometry file.
     See Also
     --------
     write_surface
     """
+    try:
+        import nibabel as nib
+        has_nibabel = True
+    except ImportError:
+        has_nibabel = False
+    if has_nibabel and LooseVersion(nib.__version__) > LooseVersion('2.1.0'):
+        return nib.freesurfer.read_geometry(fname, read_metadata=read_metadata)
```

Comment:
Yes it's fixed in https://github.com/nipy/nibabel/pull/461#issuecomment-228140497